packages/tupaia-web/src/utils/useRemoveParams.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export const useRemoveParams = (paramsToRemove?: string[]) => {
```

Comment:
```suggestion
export const removeUrlSearchParams = (paramsToRemove?: string[]) => {
```
The `use` prefix is usually only used for custom react hooks which is a function that has a hook inside of it. Since there isn't one in this util it's probably best not to use the prefix. Also I always get mixed up between url params and query string params so it would be good to be explicit with the name of the util.